src/Microsoft.Health.Fhir.SqlServer/Features/Storage/SqlServerFhirDataStore.cs

```diff
@@ -283,12 +287,31 @@ public async Task HardDeleteAsync(ResourceKey key, CancellationToken cancellatio
 
         public async Task UpdateSearchParameterIndicesBatchAsync(IReadOnlyCollection<ResourceWrapper> resources, CancellationToken cancellationToken)
```

Comment:
I plan to rename this method along with `UpdateSearchIndexForResourceAsync` in [AB#80106](https://microsofthealth.visualstudio.com/Health/_workitems/edit/80106).